config/database.php

```diff
@@ -96,13 +96,17 @@
         'client' => 'predis',
 
         'default' => [
+            'scheme' => env('REDIS_SCHEME', 'tcp'),
+            'path' => env('REDIS_PATH', '/var/run/redis/redis.sock'),
```

Comment:
This should probably be changed to `/run/redis/redis.sock` as most modern OSes symlink `/var/run` to `/run`.